program/Makefile

```diff
@@ -1,2 +1,995 @@
-PDFGrader.exe: src/main/System.vala src/main/QuestionSet.vala src/main/Question.vala src/main/Mark.vala src/main/Save.vala
-	valac -o PDFGrader.exe --pkg=gtk+-3.0 --pkg=poppler-glib --pkg cairo --pkg gee-0.8 --pkg glib-2.0 --pkg libxml-2.0 --pkg posix src/main/System.vala src/main/QuestionSet.vala src/main/Question.vala src/main/Mark.vala src/main/Save.vala src/main/Lock.vala src/main/Export.vala
+# Makefile.in generated by automake 1.15.1 from Makefile.am.
```

Comment:
Why are we replacing the Makefile with an auto-generated one?

Reply:
Really not sure how this snuck its way in, don't think anyone here made changes to this file on purpose. Is there a way to revert just this one file to the source branch?

Reply:
Ran 'git checkout develop -- program/Makefile' to revert the makefile to what it is on develop. Lmk if this isn't correct/should be done in a different way.